frontend/src/utils/index.js

```diff
@@ -643,3 +643,26 @@ export function omitKeysWithSuffix (obj, suffix) {
   const keys = Object.keys(obj).filter(key => key.endsWith(suffix))
   return omit(obj, keys)
 }
+
+export function parseNumericalAbbreviation (abbreviatedNumber) {
+  const regex = /(\d+(?:\.\d+)?)(k|M|B)?/i
+  const match = abbreviatedNumber.match(regex)
+
+  if (!match) {
+    return null
+  }
+
+  const number = parseFloat(match[1])
+  const suffix = match[2]
```

Comment:
If the input is `x1` we return 1.
If the input is `1.` we return null
if the input is `1km` we return 1000 
Wouldn't it be better to return NaN if parsing fails. The reason why we return `null` is that we want to use it directly in `min` and `max` functions. That only works with null. 
```suggestion
  const [, number, suffix] = /^(\d+(?:\.\d*)?)([kmbt]?)$/i.exec(abbreviatedNumber) ?? []
  if (!number) {
    return null
  }
```